Dockerfile

```diff
@@ -1,15 +1,17 @@
-FROM centos:7
+FROM centos:8
 
-RUN yum install -y epel-release gcc git krb5-devel python-devel python2 python3
-RUN yum install -y python-pip
-RUN pip install --upgrade pip ; pip install --upgrade setuptools
-RUN pip install koji
-RUN pip3 install tox twine setuptools wheel codecov
-RUN curl -Ls https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 --output jq \
-    && chmod +x jq && mv jq /usr/local/bin/jq
+RUN yum install -y epel-release python2 python36
+RUN yum install -y gcc git jq krb5-devel openssl-devel libcurl-devel rpm-devel \
+    python{2,3} python{2,3}-{devel,pip}
+
+# those environment variables are required by installing pycurl, koji, and rpkg with pip
+ENV PYCURL_SSL_LIBRARY=openssl RPM_PY_SYS=true
 
-RUN pip install rh-doozer==0.5.17 rh-elliott==0.2.12 rh-ocp-build-data-validator==0.0.8
+RUN pip2 install koji
+RUN pip3 install tox twine setuptools wheel codecov
 
+RUN pip2 install rh-doozer==0.5.17 rh-elliott==0.2.12 rh-ocp-build-data-validator==0.0.8
+RUN pip2 install pycurl rpkg
```

Comment:
Minor tweak: invert lines `13` and `14`, since we'll update line `13` much more often than `14`, so we can rely on cached layers ;D

Reply:
Good point. I finally decided to remove line 14 and let them to be installed by dependencies.